test/hack/resource/pkg/resourcetypes/resourcetypes.go

```diff
@@ -27,6 +27,7 @@ const (
 	karpenterTestingTag        = "testing/cluster"
 	k8sClusterTag              = "cluster.k8s.amazonaws.com/name"
 	githubRunURLTag            = "github.com/run-url"
+	resourceRegionTag          = "topology.kubernetes.io/region"
```

Comment:
This an upstream label. Should we just use that constant rather than copying the tag into our own environment?